### PR TITLE
seeds

### PR DESCRIPTION
**Pull request by Naokimi**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,62 @@
...
+(1..5).to_a.each do |number|
+  WorkGroup.create!(name: "Group #{number}", video_call_code: 'abc', classroom: Classroom.first)
+end
+
+p 'assigning students to workgroups'
+
+students = Student.all
+work_groups = WorkGroup.all
+students.each_with_index do |student, index|
+  StudentWorkGroup.create!(student: student, work_group: work_groups[index % 4])
```
Comment (AnnK89) [inline db/seeds.rb]: Just wanna understand why do we use [index % 4] here?

**Review by AnnK89 — APPROVED** [review]

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,63 @@
...
+StudentWorkGroup.destroy_all
+WorkGroup.destroy_all
+Classroom.destroy_all
+Student.destroy_all
+Teacher.destroy_all
+
+p 'creating teacher'
+
+Teacher.create!(name: Faker::Name.name)
+
```
Comment (JulienErgan) [inline db/seeds.rb:22]: `p 'Finished creating #{Teacher.count} Teachers'`

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,63 @@
...
+p 'creating students'
+
+20.times do
+  Student.create!(name: Faker::Name.name)
+end
+
```
Comment (JulienErgan) [inline db/seeds.rb:30]: p 'Finished creating #{Student.count} students'

File: db/seeds.rb [hunk 4]
```diff
@@ -5,3 +5,63 @@
...
+p 'creating classroom'
+
+Classroom.create!(teacher: Teacher.first)
+
```
Comment (JulienErgan) [inline db/seeds.rb:36]: p 'Finished creating #{Classroom.count} classroom'

File: db/seeds.rb [hunk 5]
```diff
@@ -5,3 +5,63 @@
...
+p 'creating work groups'
+
+(1..5).to_a.each do |number|
+  WorkGroup.create!(name: "Group #{number}", video_call_code: 'abc', classroom: Classroom.first)
+end
+
```
Comment (JulienErgan) [inline db/seeds.rb:44]: p 'Finished creating #{WorkGroup.count} work groups'

File: db/seeds.rb [hunk 6]
```diff
@@ -5,3 +5,63 @@
...
+p 'assigning students to workgroups'
+
+students = Student.all
+work_groups = WorkGroup.all
+students.each_with_index do |student, index|
+  StudentWorkGroup.create!(student: student, work_group: work_groups[index % 4])
+end
+
```
Comment (JulienErgan) [inline db/seeds.rb:54]: p 'Finished assigning students to work groups'

File: db/seeds.rb [hunk 7]
```diff
@@ -5,3 +5,63 @@
...
+    correct_content: {
+      headers: ['japanese', 'english', 'past', 'past participle'],
+      example: ['hajimeru', 'begin', 'began', 'begun'],
+      1 => ['hashiru', 'run', 'ran', 'run'],
+      2 => ['iu', 'say', 'said', 'said']
+    }.to_json,
+    work_group: work_groups[number]
+  )
+end
+
```
Comment (JulienErgan) [inline db/seeds.rb:76]: p 'Finished creating #{Worksheet.count} worksheets'

**Review by JulienErgan — CHANGES_REQUESTED** [review]
Added comments to help tracking potential errors when running rails db:seed
Feel free to reject those suggestions if you feel they're not necessary

**Pull request merged**